.github/BEFORE_YOUR_FIRST_ISSUE.md

```diff
@@ -90,20 +135,47 @@ service_role key: eyJh......
 
 ```
 You can view these details again at any time by running the following command:
+
+<details>
+<summary><strong>macOS / Linux</strong></summary>
 ```bash
 supabase status
+```
```

Comment:
When we look at the preview, this example and the following _IOS_ script previews are not getting formatted correctly:

![Screenshot 2024-05-18 at 16 05 20](https://github.com/enBloc-org/kindly/assets/114600712/e70f3457-e5f9-4161-b58a-6a8bc8bd2aea)

I think this is because they all need to have a new line added before the start of the quotation block otherwise the markdow will not get picked up properly